### PR TITLE
add tests for intermediate dashboard tables that will be modified

### PR DESCRIPTION
**Pull request by sravfeyn**
@nickpell 
@snopoke

### CODE REVIEW

File: custom/icds_reports/tests/test_dashboard_tables.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            sort_key=[sort_key]
+        )
+
+
+class AwwIncentiveAndDailyAttendanceTest(AggregationScriptTestBase):
+    def test_aww_incetive(self):
+        for month in [date(2017, 4, 1), date(2017, 5, 1)]:
+            # this is not built in setUpModule()
+            build_incentive_report(month)
+            import pdb; pdb.set_trace()
```
Comment (sravfeyn) [inline custom/icds_reports/tests/test_dashboard_tables.py]: (Will remove this)

**Review by snopoke — COMMENTED** [review]
Wow, that's a lot of CSVs. When do these tables get populated? Is it during the module level setup?

**Comment by sravfeyn** [comment]
Yeah, all of them get built during setUpModule (via the big agg task) except aww_incentive (which I am still trying to figure out). There is a table for each combination of state and month. I am only testing 4 combinations, there are more that I am not testing.

**Comment by snopoke** [comment]
@calellowitz @emord how necessary do you think this is? This was my idea but I'm having second thoughts seeing how many CSV files there are. This could make updating tests even harder.

**Comment by emord** [comment]
I don't think its necessary which is why I never wrote tests for these. I believe that the higher level test of UCR table -> Aggregate table should be more than sufficient to test out the intermediate tables.

These tests are already a pretty big pain to update, so i'd vote for having these tested by proxy

**Comment by sravfeyn** [comment]
Closing per https://github.com/dimagi/commcare-hq/pull/23439#issuecomment-467915277

**Pull request closed without merging**